trust-center-agent/src/main/java/care/smith/fts/tca/consent/configuration/GicsFhirConfiguration.java

```diff
@@ -29,7 +29,7 @@ public class GicsFhirConfiguration {
       List.of("allConsentsForDomain", "allConsentsForPerson");
 
   @NotBlank String baseUrl;
-  int defaultPageSize = 50;
+  @Deprecated int defaultPageSize = 50;
```

Comment:
I think we should make this optional, so changing it to an Integer and then check if it is set - print a warning in this case. Add a qualifier for the bean, then we inject both and the non-deprecated takes precedence when both are set.